iubenda/Wappalyzer
# Make project point to Wappalyze Module

A simple solution to update wappalyer module which is basically an automated way to download the latest version of the module from the npm registry ( i.e: [https://registry.npmjs.org/wappalyzer/-/wappalyzer-5.4.18.tgz]() )
and execute a merge. In order to this, the current wappalazer repo on iubenda should point to the new modified wappalyzer module which can be updated. 
To launch the update process simply run `npm run update` from project's directory. 
the script will: 

1. Fetch the latest wappalyzer module in zipped format
2. Uncompress the zipped module into the current dir
3. Create new branch called "update-[module's version]"
4. Copy the new module content to the new branch
5. Checkout and merge into master branch

The module should now reflect the latest changes, any outputs of the process are printed on stdout.
_A manual review is still required in case of merge conflicts._
